### ISSUE TITLE
Playtime still going up when vanished

### ISSUE THREAD

**Issue by Recliness** [Zrips/CMI#8618]
### Description of Issue

As you can see in the image below, my playtime is still going up, even though CMI is detecting that I am vanished.

![image](https://github.com/Zrips/CMI/assets/48137058/9f636041-4d6c-445b-a7c4-d2d3d1836584)

### Version Information

```shell
[13:33:59 INFO]: --------------------------------------------------
[13:33:59 INFO]: CMI: 9.7.0.4 Velocity CMIB  MySQL
[13:33:59 INFO]: CMILib: 1.4.6.1 
[13:33:59 INFO]: Server: Purpur(2169) 1.20.4-R0.1-SNAPSHOT-
[13:33:59 INFO]: CMI economy: Enabled Vault: 1.7.3-CMI CMI Chat: Enabled 
[13:33:59 INFO]: Modules -> 52 enabled 7 disabled: spawnerProximity, tablist, ranks, moneyCheque, holograms, spawners, spawnerCharge
[13:33:59 INFO]: --------------------------------------------------
```

### Relevant Config Sections

```yaml
config.yml lines 1719-1743


Vanish:
  # Defines default states of vanish edit options for players
  # This will not have any effect if player already edited his vanish mode with vanishedit command
  Defaults:
    damageToEntity: false
    playerDamage: false
    itemPickup: false
    mobAggro: false
    interaction: false
    noisyChest: false
    informOnLeave: false
    informOnJoin: false
    nightVision: true
    bossbar: true
    afkcommands: false
    PrivateMessages: false
    relogDisable: false
    noMessages: false
    fakeJoinLeave: false
    mobSpawning: false
    stopPlaytime: true
    sleepIgnore: true
    joinVanished: false
    deathMessages: false
    hookPlayers: false
```

### Relevant Plugins

PremiumVanish 2.9.9

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by Zrips** [comment]
Double check that your player actually has enabled stopPlaytime, you can do that with **/cmi vanishedit** gui or dirrectly with **/vanishedit Zrips stopplaytime true** while config file defaults are just defaults and will not have effect if you already modified your vanish settings ingame at some point.

**Comment by ReclinesMC** [comment]
> Double check that your player actually has enabled stopPlaytime, you can do that with **/cmi vanishedit** gui or dirrectly with **/vanishedit Zrips stopplaytime true** while config file defaults are just defaults and will not have effect if you already modified your vanish settings ingame at some point.



Yep! It's turned on to stop playtime. For more context (because apparently I didn't give much in the original) I'm using PremiumVanish. 

![image](https://github.com/Zrips/CMI/assets/48137058/581b8475-0afc-4717-a018-ac4684fbe837)
![image](https://github.com/Zrips/CMI/assets/48137058/6aeccb0c-52cb-4d99-9753-75393401c208)

Weirdly enough, even though it says that I am vanished in /info, it doesn't think I'm vanished in the gui.

**Comment by ReclinesMC** [comment]
Heyo @Zrips, did that latest update fix this issue?

**Comment by Zrips** [comment]
PremiumVanish would be separate vanish handling on its own, so CMI will not really do much here. All of those vanish features are based on player going into vanish mode by using CMI vanish as it needs appropriate triggers when player enters and leaves this mode

**Comment by ReclinesMC** [comment]
> PremiumVanish would be separate vanish handling on its own, so CMI will not really do much here. All of those vanish features are based on player going into vanish mode by using CMI vanish as it needs appropriate triggers when player enters and leaves this mode

So what you're saying is that I literally have to disable players' abilities to do /playtime... just because I use a third party vanish plugin?

**Comment by Recliness** [comment]
damn
